program/src/oracle/oracle.c

```diff
@@ -490,6 +490,9 @@ static uint64_t upd_price( SolParameters *prm, SolAccountInfo *ka )
   // update aggregate price as necessary
   if ( sptr->slot_ > pptr->agg_.pub_slot_ ) {
     upd_aggregate( pptr, sptr->slot_, sptr->unix_timestamp_ );
+    sol_log("Instruction: UpdateAggregatePrice");
```

Comment:
the wording of this log statement is a little confusing to me because the instruction is actually UpdatePrice, and this just happened to be the first invocation of that within the slot, so aggregation was triggered.

Reply:
and also maybe the log below should be agg_price, agg_conf, etc?